src/solvers/sat/satcheck_minisat2.h

```diff
@@ -23,11 +23,23 @@ class Solver; // NOLINT(readability/identifiers)
 class SimpSolver; // NOLINT(readability/identifiers)
 }
 
+class optionst;
+
+struct satcheck_minisat_infot final
+{
+  /// Parse satcheck options
+  /// \param options Options object
+  /// \return An instance of satcheck_minisat_info object
+  static satcheck_minisat_infot parse(const optionst &options);
+  size_t random_seed=91648253; // Default copied from minisat2
+  double random_var_freq=0.;
```

Comment:
As these have default values, wouldn't it be possible to follow @kroening consideration to maintain the existing interface of solvers that can be default-constructed without such options?